runtime/src/append_vec.rs

```diff
@@ -155,22 +156,53 @@ pub struct StoredMeta {
     pub pubkey: Pubkey,
 }
 
+bitflags! {
+    #[derive(Serialize, Deserialize, Default)]
+    pub struct AccountFlags: u8 {
+        const NONE = 0b00000000;
```

Comment:
We don't really need to define `NONE` explicitly - it's actually a bit confusing imo since it's not really a flag.

The typical way to have flags set is to use the `empty()` fn, i.e. `AccountFlags::empty()`.